### ISSUE TITLE
About  Generator loss

### ISSUE THREAD

**Issue by YuweiZ** [shaohua0116/SSGAN-Tensorflow#10]
# Generator loss
            g_loss = tf.reduce_mean( tf.log(d_fake[:, -1]))

OR 
           g_loss = tf.reduce_mean(-tf.log(d_fake[:, -1]))

**Comment by shaohua0116** [comment]
The first one.
